src/azureExtensions.ts

```diff
@@ -92,6 +92,19 @@ export const azureExtensions: IAzExtMetadata[] = [
             AzExtResourceType.SpringApps,
         ],
         reportIssueCommandId: 'springApps.reportIssue'
+    },
+    {
+        name: 'vscode-azurelogicapps',
+        publisher: "ms-azuretools",
+        label: 'Logic Apps',
+        resourceTypes: [
+            AzExtResourceType.LogicApp,
+        ],
+        tutorial: {
+            label: localize('createLogicApp', 'Create a standard logic app'),
+            url: 'https://go.microsoft.com/fwlink/?linkid=2222287'
```

Comment:
Could you make this an aka.ms link? It's helpful in case this url ever changes so we can fix it without having to release.

Reply:
Sure, I will update the link